git-version-gen

```diff
@@ -2,7 +2,7 @@
 
 # usage: git-version-gen [srcdir]
 
-DEF_VER="v0.7.GIT"
+DEF_VER="v0.8.GIT"
```

Comment:
This is causing the CI build. Probably because .GIT won't match the REGEX in the cmake script.